client/components/RowA.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-function RowA({ firstName, lastName, userImage }) {
```

Comment:
might have made sense from a data standpoint to have a field titled: 'username' in your database especially since most people don't want their real name to be displayed as the author of the review. Doesn't really matter though! Thoughts for future developments.